# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/gpm/liblow-silent.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/lib/liblow.c.orig	2004-10-02 06:30:04.602331640 +0200
+++ ./src/lib/liblow.c	2004-10-02 06:30:57.756251016 +0200
@@ -201,7 +201,9 @@
 
    option.consolename = NULL;
 
+#if 0
    gpm_report(GPM_PR_DEBUG,"VC: %d",flag);
+#endif
 
    /*....................................... First of all, check xterm */
 
@@ -369,7 +371,9 @@
 
   /*....................................... Error: free all memory */
    err:
+#if 0
    gpm_report(GPM_PR_ERR,"Oh, oh, it's an error! possibly I die! ");
+#endif
    do {
       new=gpm_stack->next;
       free(gpm_stack);
